src/iceqube/queue.py

```diff
@@ -0,0 +1,110 @@
+from iceqube.classes import Job
+from iceqube.storage import Storage
+
+DEFAULT_QUEUE = "ICEQUBE_DEFAULT_QUEUE"
+
+
+class Queue(object):
+    def __init__(self, app=DEFAULT_QUEUE, storage_path=None):
+        if storage_path is None:
+            raise ValueError('Storage path must be defined')
+        self.storage = Storage(app, app, storage_path)
+
+    def __len__(self):
+        return self.storage.count_all_jobs()
+
+    @property
+    def job_ids(self):
+        return [job.job_id for job in self.storage.get_all_jobs()]
+
+    @property
+    def jobs(self):
+        """
+        Return all the jobs scheduled, queued, running, failed or completed.
+        Returns: A list of all jobs.
+
+        """
+        return self.storage.get_all_jobs()
+
+    def enqueue(self, func, *args, **kwargs):
```

Comment:
Is this the official API for iceqube now?

Reply:
Yeah, this is parallel to RQ's API.